framework/include/constraints/RealMortarConstraint.h

```diff
@@ -0,0 +1,185 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef REALMORTARCONSTRAINT_H
```

Comment:
What's 'Real' about this?  Is it Real as in really being a mortar constraint, or is this referring to Real numbers?

Reply:
It's really a mortar constraint. This name will change; it will likely just replace the existing `MortarConstraint` class